Source/Outposts/Outpost/Outpost_Deliveries.cs

```diff
@@ -12,16 +12,22 @@ public partial class Outpost
     {
         public void Deliver(IEnumerable<Thing> items)
         {
-            var map = Find.Maps.Where(m => m.IsPlayerHome).OrderByDescending(m => Find.WorldGrid.ApproxDistanceInTiles(m.Parent.Tile, Tile)).First();
-
+            var things = items.ToList();            
+            var map = deliveryMap != null ? deliveryMap :Find.Maps.Where(m => m.IsPlayerHome).OrderByDescending(m => Find.WorldGrid.ApproxDistanceInTiles(m.Parent.Tile, Tile)).FirstOrDefault();
```

Comment:
Is this necessary with the `SpawnSetup` one? I think one or the other would suffice. I vote this one.
```suggestion
            var map = deliveryMap != null ? deliveryMap : Find.Maps.Where(m => m.IsPlayerHome).OrderBy(m => Find.WorldGrid.ApproxDistanceInTiles(m.Parent.Tile, Tile)).FirstOrDefault();
```